pkg/bundle/bundle.go

```diff
@@ -97,7 +104,6 @@ func (b *Bundle) GenerateBundlePublishBody(srcDir string) (restclient.PublishPos
 }
 
 func (b *Bundle) IsInfrastructure() bool {
-	// a Deprecation warning is printed in the bundle parse function
```

Comment:
I couldn't find this deprecation warning anywhere. Plus, we've replaced `bundle` with `infrastructure` in all our bundles.